spec/conventions.html

```diff
@@ -92,12 +92,12 @@ <h1>Well-Known Intrinsic Objects</h1>
           <td>The `Intl.Segmenter` constructor (<emu-xref href="#sec-intl-segmenter-constructor"></emu-xref>)</td>
         </tr>
         <tr>
-          <td>%IntlSegmentIteratorPrototype%</td>
+          <td>%Intl.SegmentIteratorPrototype%</td>
```

Comment:
Per [Well-Known Intrinsic Objects](https://tc39.es/ecma262/#sec-well-known-intrinsic-objects), the syntax `%Intl.SegmentIteratorPrototype%` means that the `"SegmentIteratorPrototype"` property of the intrinsic object `%Intl%` is accessed. This wrong because there's no `Intl.SegmentIteratorPrototype` property and therefore the intrinsic name `%IntlSegmentIteratorPrototype%` has to be kept.